src/CoreBundle/ResourceCollection/ResourceCollectionInformationService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\ResourceCollection;
+
+use ChameleonSystem\CoreBundle\Interfaces\ResourceCollectorInterface;
+
+class ResourceCollectionInformationService implements ResourceCollectionInformationServiceInterface
+{
+    /**
+     * @var ResourceCollectorInterface
+     */
+    private $resourceCollector;
+
+    public function __construct(ResourceCollectorInterface $resourceCollector)
+    {
+        $this->resourceCollector = $resourceCollector;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isActive(): bool
```

Comment:
-> sounds like "resource collection is activated?"

-> isRunning()?

Reply:
no - in fact, wenn resource collection is running, this method will return false. See documentation in `ResourceCollectionInformationServiceInterface::isActive` when the method will return true.